src/Repository/StepRepository.php

```diff
@@ -19,6 +20,18 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, Step::class);
     }
 
+    /**
+     * @throws NonUniqueResultException
+     */
+    public function findOneByUuid(string $uuid): ?Step
+    {
+        return $this->createQueryBuilder('S')
+            ->andWhere('S.uuid = :uuid')
```

Comment:
Pourquoi utiliser un "AndWhere" alors que tu peux utiliser un ```where``` ? Le And risque de bloquer quoi

Reply:
```suggestion
            ->where('S.uuid = :uuid')
```